authentication/views.py

```diff
@@ -84,6 +87,20 @@ def get_serializer_cls(self):
         """Return the serializer cls"""
         return RegisterEmailSerializer
 
+    def post(self, request):
+        """ Verify recaptcha response before proceeding """
+        if settings.RECAPTCHA_SITE_KEY:
+            r = requests.post(
+                "https://www.google.com/recaptcha/api/siteverify?secret={key}&response={captcha}".format(
+                    key=escape(settings.RECAPTCHA_SECRET_KEY),
```

Comment:
I think you need to use [urllib.parse.quote](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote) here instead of `html.escape`

Reply:
replaced `escape` with `quote`